index.js

```diff
@@ -20,14 +20,24 @@ class PinoTransform extends Transform {
   }
 
   _transform (chunk, encoding, callback) {
-    const { column, passThrough } = this.opts
+    const { column, passThrough, wrapNonJson } = this.opts
     const content = chunk.toString('utf-8')
 
+    try {
+      JSON.parse(content)
+    } catch (e) {
+      if (wrapNonJson) {
+        const fixedcontent = `{"time":${Date.now()},"msg":"${content}"}`
+        buffer.push({ [column]: fixedcontent })
+        return callback(null, passThrough ? `${chunk}\n` : null)
+      }
+    }
     buffer.push({ [column]: content })
```

Comment:
A more compact version could be this

```js
    let content = chunk.toString('utf-8')

    try {
      JSON.parse(content)
    } catch {
      content = `{"time":${Date.now()},"msg":"${content}"}`
    }

    buffer.push({ [column]: content })
```

And also, in this case, I believe we can have this the default behaviour and not use the flag. This validation should have been there in the first place.

Reply:
Hi! I would suggest keeping the flag since 2 use cases are possible:
1) Database uses a json/jsonb column. All messages must be valid json. Simple text messages need to be converted to json first.
2) Database uses a string/varchar column. Non-json messages could be stored as they are, without need to be wrapped as json.
Maybe the user will not expect the transport to renecode the messages unless it is explicitly stated in the command line parameters?

Reply:
Originally I did not expect users of this library to use non-jsonb column. However, if you think that's a real use case, I don't mind keeping the flag.

Could you update the `Usage` section in readme with the new flag. You can get the full output with `--help` param.